lib/src/utf16.dart

```diff
@@ -8,15 +8,15 @@ import 'dart:typed_data';
 
 import 'package:ffi/ffi.dart';
 
-/// [Utf16] implements conversion between Dart strings and null-terminated
+/// [Utf16] implements conversion between Dart strings and zero byte terminated
 /// UTF-16 encoded "char*" strings in C.
 ///
-/// [Utf16] is respresented as a struct so that `Pointer<Utf16>` can be used in
+/// [Utf16] is represented as a struct so that `Pointer<Utf16>` can be used in
 /// native function signatures.
 class Utf16 extends Struct {
-  /// Convert a [String] to a Utf16-encoded null-terminated C string.
+  /// Convert a [String] to a UTF-16 encoded zero byte terminated C string.
   ///
-  /// If 'string' contains NULL bytes, the converted string will be truncated
+  /// If 'string' contains zero bytes, the converted string will be truncated
   /// prematurely. Unpaired surrogate code points in [string] will be preserved
   /// in the UTF-8 encoded result. See [Utf8Encoder] for details on encoding.
```

Comment:
Should probably say UTF-16 here.
(You don't have to fix all existing typos, I'm just mentioning it so someone, like @dcharkes, can fix it later). :wink: